<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1418</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
box-shadow: 0 0 1rem black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>741</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>741</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>98</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
background-color: rgb(0, 0, 127);
color:white;
}</string>
     </property>
     <property name="text">
      <string>Chào,</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
background-color: rgb(0, 0, 127);
color:white;
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>640</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Báo Cáo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>590</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Môn Học</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Điểm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Khối</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Lớp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Quyền Hạn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Chức Vụ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Nhân Viên</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Giáo Viên</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHocSinh">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Học Sinh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_21">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Tài Khoản</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_22">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>690</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
	background-color: rgb(0, 0, 127);
	border-left: 2px white rgb(255, 255, 255);
	border-radius:2px;
border-bottom:2px solid white;
	font: 16pt &quot;Tahoma&quot;;
}
QPushButton:hover{
color:Black;
border-bottom:1px solid black;
background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Đăng Xuất</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>1221</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackeWidget{
background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>781</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Tahoma&quot;;
color: rgb(170, 170, 127);</string>
      </property>
      <property name="text">
       <string>Chào bạn! Chúc bạn một ngày làm việc hiệu quả </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>741</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1051</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>QUẢN LÝ HỌC SINH</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>20</y>
        <width>141</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>CHỨC NĂNG </string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 16pt &quot;Tahoma&quot;;
border-radius:15px;
border:1px solid green;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
	font: 18pt &quot;Tahoma&quot;;
}
</string>
       </property>
       <property name="text">
        <string>Thêm </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 16pt &quot;Tahoma&quot;;
border-radius:15px;
border:1px solid blue;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	font: 18pt &quot;Tahoma&quot;;
	background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 85, 255, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
       </property>
       <property name="text">
        <string>Sửa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 16pt &quot;Tahoma&quot;;
border-radius:15px;
border:1px solid red;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	font: 18pt &quot;Tahoma&quot;;
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.028, stop:0 rgba(189, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
}
</string>
       </property>
       <property name="text">
        <string>Xóa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 16pt &quot;Tahoma&quot;;
border-radius:15px;
border:1px solid yellow;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	font: 18pt &quot;Tahoma&quot;;
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 127, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
       </property>
       <property name="text">
        <string>Nhập Excel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 16pt &quot;Tahoma&quot;;
border-radius:15px;
border:1px solid red;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 85, 0, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	font: 18pt &quot;Tahoma&quot;;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
       </property>
       <property name="text">
        <string>Xuất Excel</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>1201</width>
        <height>381</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1418</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../img/login.qrc"/>
 </resources>
 <connections/>
</ui>
